### ISSUE TITLE
Timeout on pg_dumpall

### ISSUE THREAD

**Issue by coderanger** [chef/chef_backup#19]
The tar strategy uses the default 10 minute timeout for pg_dumpall. With lots of reporting data it is easy to blow past this. Should either be set higher or made overridable.

**Comment by ryancragun** [comment]
This should be resolved by https://github.com/chef/chef_backup/commit/bd29c56a0adb93c1443201088024b09f42b508a2. 

Postgresql DB operations are now run with a default timeout of 60 minutes.  If you wish to change it (or any other PG option) you can use the pg_options switch, eg:

`chef-server-ctl restore my_backup.tgz --pg-options '-c statement_timeout=66666666666666'`

You can track the upstream PR that bundles this fix in the Chef Server here https://github.com/chef/chef-server/pull/770

**Comment by coderanger** [comment]
@ryancragun I don't think that is correct. You set a postgres-level timeout, but the root problem was the shellout timeout.

**Comment by coderanger** [comment]
Also the default value of `statement_timeout` is 0 meaning it is disabled so actually you just made things worse.

**Comment by coderanger** [comment]
Pinging again to re-open this as it was not fixed.

**Comment by pburkholder** [comment]
@coderanger Reopening....
